platform/android/export/export_plugin.cpp

```diff
@@ -2618,10 +2618,6 @@ bool EditorExportPlatformAndroid::has_valid_export_configuration(const Ref<Edito
 			err += template_err;
 		}
 	} else {
-#ifdef ANDROID_ENABLED
-		err += TTR("Gradle build is not supported for the Android editor.") + "\n";
-		valid = false;
-#else
```

Comment:
The matching `#endif` needs to be removed as well.

Reply:
I removed the `#endif` in my commit. They'll get squashed together in the end